Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM python:3.8-alpine
-EXPOSE 8888
+FROM python:3.8-slim
+WORKDIR /mclocations
 
-WORKDIR /usr/src/app
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
Why --no-cache-dir? I pulled this from the documentation from the [docker repo](https://hub.docker.com/_/python) I figured this was important if the maintainers included it.

Reply:
So I think `--no-cache-dir` might be a good idea? Let me check the docs... https://pip.pypa.io/en/stable/reference/pip_cache/#cmdoption-no-cache-dir
> --no-cache-dir
> Disable the cache.

Oh, yes.

Reply:
https://pip.pypa.io/en/stable/reference/pip_install/#caching

Sounds like this might be a space saving technique. I actually started off by creating a multistage docker file with a build stage, but dropped it since this is really just a script (I'm not making a wheel or anything to transfer over).